src/runtime/audience-action-flow.js

```diff
@@ -163,6 +164,8 @@ export class AudienceActionFlow {
     } else {
       this.showFailedOptedInToast_();
     }
+    const now = Date.now().toString();
```

Comment:
So the only granularity we really care about is seconds. I feel like maybe we just store and use that value and just make sure our documentation indicates that